### PR TITLE
OOVPA : Finalize D3D8LTCG Database for 4039 and newer

### PR DESCRIPTION
**Pull request by jarupxx**
Actually, SetVertexShaderInput_1036 is SetPalette in 5120

### CODE REVIEW

**Comment by jarupxx** [comment]
It's okay. I was investigaed at the correct SetVertexShaderInput_1036，but does not exist.

File: OOVPADatabase/D3D8LTCG.OOVPA.inl [hunk 1]
```diff
@@ -112,30 +114,33 @@ OOVPATable D3D8LTCG_OOVPAV2[] = {
...
 	REGISTER_OOVPAS(D3DDevice_SetLight, 1024, 1048),
-	REGISTER_OOVPAS(D3DDevice_SetPalette, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_SetPalette, 1024, 1036, 1048),
 	REGISTER_OOVPAS(D3DDevice_SetPalette_4, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShader, 1024, 1036, 1048),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderConstant, 1024, 1036),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderConstant_4, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderProgram, 1024),
-	REGISTER_OOVPAS(D3DDevice_SetPixelShader_0, 2024, 2036, 2048),
+	REGISTER_OOVPAS(D3DDevice_SetPixelShader_0, 2024, 2036, 2048, 2060, 2072),
```
Comment (RadWolfie) [inline OOVPADatabase/D3D8LTCG.OOVPA.inl]: @jarupxx, I can't believe I missed this. Where is D3DDevice_SetPixelShader_0_2060 OOVPA at? I don't see it in this pull request.

Comment (jarupxx) [reply OOVPADatabase/D3D8LTCG.OOVPA.inl]: It was 3911 that leave it in this pull request. I wish I had been more careful.

File: OOVPADatabase/D3D8LTCG.OOVPA.inl [hunk 2]
```diff
@@ -163,18 +168,20 @@ OOVPATable D3D8LTCG_OOVPAV2[] = {
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_StencilFail, 1024, 1036, 1048, 1060, 1072, 1084),
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_TextureFactor, 1036),
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_TwoSidedLighting, 1024, 1036, 1048, 1060, 1072, 1084),
-	REGISTER_OOVPAS(D3DDevice_SetRenderState_VertexBlend, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_SetRenderState_VertexBlend, 1024, 1036, 1048),
```
Comment (RadWolfie) [inline OOVPADatabase/D3D8LTCG.OOVPA.inl]: And still having failed to compile, D3DDevice_SetRenderState_VertexBlend_1048 is also missing. Hopefully, this is the last one.

File: OOVPADatabase/D3D8LTCG.1.0.4039.inl [hunk 3]
```diff
@@ -951,8 +974,8 @@ OOVPA_END;
 // ******************************************************************
 // * D3DDevice_SetStreamSource
 // ******************************************************************
-//7406810300000800
-OOVPA_XREF(D3DDevice_SetStreamSource, 1040, 1+10,
+//7406810300000800 ...C20800
+OOVPA_XREF(D3DDevice_SetStreamSource_8, 2040, 1+10,
```
Comment (RadWolfie) [inline OOVPADatabase/D3D8LTCG.1.0.4039.inl]: Is this suppose to be D3DDevice_SetStreamSource_8_1040 or D3DDevice_SetStreamSource_8_2040?

**Comment by RadWolfie** [comment]
Well, three things are only thing causing problem. At least one is already fixed.

File: OOVPADatabase/D3D8LTCG.1.0.4039.inl (hunk 3 above)
Comment (PatrickvL) [reply OOVPADatabase/D3D8LTCG.1.0.4039.inl]: Also, something for in the readme : how are these codes (1040, 2040, etc) calculated?

Comment (jarupxx) [reply OOVPADatabase/D3D8LTCG.1.0.4039.inl]: LTCG specific OOVPA assign to 2000 number. It is considered to be D3DDevice_SetStreamSource_8_2040.

I also those code related into HLEIntercept.cpp, I used a code reminiscent of XDK revision. XXYY: (XX Standard or LTCG specific, YY XDK revision)
I agree to change to understandable code, If you hope.

**Pull request merged**